.github/workflows/e2e.yaml

```diff
@@ -0,0 +1,28 @@
+name: e2e tests
+
+on:
+  schedule:
+    - cron: "0 */12 * * *"
```

Comment:
Good question! Several times we had broken stacks due to various issues but I guess those issues all originate from PR's (or at least they should).

What if we do it on all PR and on direct pushes to master... As those do occur from time to time?

Reply:
If we have it both for PR and push events, then we would be running it twice for a given PR once it's merged. 🤔 

We should rather avoid pushing to master directly I think.

Reply:
Yes, we should :-)

Reply:
It's not as simple 😅 Unless you force PRs into being sequential you have to run tests on master as well. We have this in dhis2-core. Imagine PR 1 created first, then PR 2. Both contain independent changes. PR 2 changes an interface PR 1 relies on, which actually breaks changes in PR 1. If you merge them in any order without updating the PRs onto latest master you will have a broken master and not notice it right away.

So either 1. you run tests on master and PRs. Or 2. you force branches to be up to date before they can be merged. 2. is super painful if you have many devs like in dhis2-core. So for us 2. might be ok. As the PR check duration increases it might not anymore.

Reply:
Very good point, @teleivo. Thanks!

Reply:
On PRs and pushes to master it is!